PowerShell Contributing.md

```diff
@@ -0,0 +1,399 @@
+You may be able to help the filer of the bug with that.
+
+> Tip: consider 'Watching' the github repo.
+You'll get a feel for what is going on daily.
+
+### 3. Comment on language change proposals
+
+> Requires
+> 
+> - GitHub account
+
+Bigger changes or more complicated features/ideas go through the Request For Change (RFC) process.
+Anyone can submit a prosal and  request comments from the community and the team.
+Typically, an RFC will remain open for two months, to give everyone the opportunity to comment.
+Then the Committee will then make a decision on the subject.
+
+The RFC may be discussed in the monthly call if there is reason to.
+
+
+The full process is explained here: <https://github.com/PowerShell/powershell-rfc/blob/master/RFC0000-RFC-Process.md>
+
+By commenting on an RFC, you can influence how an idea is going to be implemented, if at all.
+Comments are valueable for the community and the committee to make a balanced decision, weighing all voices.
+
+### 4. Attend the monthly PowerShell community call
+
+> Requires
+> 
+> - A microphone and speakers
+> - You can even join anonymously if you like 
+> 
+> Optional
+> 
+> - MS Teams
+> - A headset
+> - A camera
+
+Every month, there is a call with PowerShell team members (committee) and community members are invited! 
+You can find the join link, as well as transcripts/youtube recordings of previous calls here: 
+<https://github.com/PowerShell/PowerShell-RFC/tree/master/CommunityCall/>
+
+Goal of the meeting is sharing roadmap/release info, and discussing RFC's.
+You can join in, ask questions and let your voice be heard.
+Eg. if there is an RFC you have commented on, you can explain your view.
+
+
+### 5. Suggest an improvement
+
+> Requires
+> 
+> - GitHub account
+
+You can also come up with ideas yourself!
+
+If you have a cool improvement for a cmdlet, a new feature or just a great idea, you can submit it as a [feature request](https://github.com/PowerShell/PowerShell/issues/new?assignees=&labels=Issue-Enhancement&template=Feature_Request.md&title=Feature+Request).
+
+The maintainers will look at your idea and let you know what they think. All genuine feature requests are handled with respect.
+So don't be afraid to share your thoughts! 
+
+Do take the time to find out if your idea has been proposed before and what was decided and why.
+You can filter on the label 'Issue-Enhancement'.
+
+
+In some cases it may be a good move to 'test' your idea against a few people first, before taking the time to fully explain what you have in mind in an issue, a great place to do so would be the [PowerShell Discord/Slack/IRC channel](link required).
+
+### 6. Contribute Help content
+
+> Requires
+> 
+> - GitHub account
+> - Knowledge of Markdown
+> 
+> Optional
+> 
+> - Git and some knowledge of it
+> - Knowledge of GitHub workflow (fork > clone > PR)
+
+You can contribute by writing, improving, expanding help files.
+The ones that are used by Get-Help.
```

Comment:
This sentence feels a bit redundant, almost. Perhaps try working it into the previous sentence?